modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -435,6 +439,9 @@ class ACASFormLSCodeValueFieldController extends ACASFormAbstractFieldController
 		if @options.autoSavePickListItem? && !@options.autoSavePickListItem
 			plOptions.autoSave = false
 
+		plOptions.autoFetch = true
+		if @options.autoFetch?
+			plOptions.autoFetch = @options.autoFetch
```

Comment:
*sort of outside pr*: Previously there was no way to pass auto fetch to the picklist controller.  At one point I thought I could turn of auto fetching of the picklists but forgot about the "selectedCode" use case where the picklist actually needs to get populated with the selected value so it needs atleast that item int he picklist. 

TLDR; I left this in as a feature but it defaults to true like before.